controlsfx-samples/src/main/java/org/controlsfx/samples/tableview2/HelloTableView2.java

```diff
@@ -0,0 +1,557 @@
+import java.util.Random;
+import javafx.beans.Observable;
+import javafx.beans.binding.Bindings;
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.IntegerProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.property.SimpleIntegerProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.FilteredList;
+import javafx.collections.transformation.SortedList;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.ContextMenu;
+import javafx.scene.control.Label;
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.SelectionMode;
+import javafx.scene.control.TableCell;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.TitledPane;
+import javafx.scene.control.cell.CheckBoxTableCell;
+import javafx.scene.control.cell.TextFieldTableCell;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Circle;
+import javafx.scene.shape.Rectangle;
+import javafx.stage.Stage;
+import javafx.util.StringConverter;
+import org.controlsfx.ControlsFXSample;
+import org.controlsfx.control.action.ActionUtils;
+import org.controlsfx.control.tableview2.TableColumn2;
+import org.controlsfx.control.tableview2.TableView2;
+import org.controlsfx.control.tableview2.actions.ColumnFixAction;
+import org.controlsfx.control.tableview2.actions.RowFixAction;
+import org.controlsfx.control.tableview2.cell.ComboBox2TableCell;
+import org.controlsfx.control.tableview2.cell.TextField2TableCell;
+import org.controlsfx.samples.Utils;
+
+/**
+ *
+ * Build the UI and launch the Application
+ */
+public class HelloTableView2 extends ControlsFXSample {
+
+    private final ObservableList<Person> data = generateData(100);
+    private TableView2Sample table;
+
+    private StackPane centerPane;
+
+    public static void main(String[] args) {
+        launch(args);
+    }
+
+    @Override
+    public String getSampleName() {
+        return "TableView2";
+    }
+
+    @Override
+    public String getSampleDescription() {
+        return "The TableView2 is an advanced JavaFX TableView control, that can "
+                + " be used as drop-in replacement control for the existing TableView, and provides"
+                + " different functionalities and use cases like row and column fixing and row header.";
+    }
+
+    @Override
+    public String getControlStylesheetURL() {
+        return "/org/controlsfx/samples/tableview2.css";
```

Comment:
This URL should return the CSS used by the Control and not the sample. Please return the following:

```
return "/org/controlsfx/control/tableview2/tableview2.css";
```

Also, update the `controlsfx/src/main/java/module-info.java` to include the following so that FXSampler can read the css:

```
opens org.controlsfx.control.tableview2 to org.controlsfx.fxsampler;
```

Reply:
done